app/code/Magento/Catalog/Controller/Adminhtml/Product/Action/Attribute/Save.php

```diff
@@ -32,23 +33,50 @@ class Save extends \Magento\Catalog\Controller\Adminhtml\Product\Action\Attribut
     protected $_catalogProduct;
 
     /**
-     * @var \Magento\CatalogInventory\Api\Data\StockItemInterfaceFactory
-     */
-    protected $stockItemFactory;
-
-    /**
```

Comment:
you can't eliminate protected variable, because it's backward incompatible change.
Somebody could extend this controller and refer to it.

It's better to mark it as @deprecated instead